### ISSUE TITLE
Crash when using segmentedPager:attributedTitleForSectionAtIndex: Delegate Method

### ISSUE THREAD

**Issue by mamouneyya** [maxep/MXSegmentedPager#63]
Hello,

I'm trying to use the delegate method: `segmentedPager(segmentedPager: MXSegmentedPager, attributedTitleForSectionAtIndex index: Int) -> NSAttributedString` to provide different formatted titles for tabs. However, I get an exception when trying to run.

Stack info:

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSConcreteAttributedString _fastCharacterContents]: unrecognized selector sent to instance 0x7f8caa1562b0'
*** First throw call stack:
(
    0   CoreFoundation                      0x0000000104759e65 __exceptionPreprocess + 165
    1   libobjc.A.dylib                     0x00000001059e8deb objc_exception_throw + 48
    2   CoreFoundation                      0x000000010476248d -[NSObject(NSObject)     doesNotRecognizeSelector:] + 205
    3   CoreFoundation                      0x00000001046af90a ___forwarding___ + 970
    4   CoreFoundation                      0x00000001046af4b8 _CF_forwarding_prep_0 + 120
    5   CoreText                            0x000000010a7baa05     _ZN19TCharStreamCFString9SetStringEO6TCFRefIPK10__CFStringE + 59
    6   CoreText                            0x000000010a75009d     _ZN23TCharStreamCFAttrStringC2EPK20__CFAttributedString + 99
    7   CoreText                            0x000000010a74fa8c     _ZN21TTypesetterAttrStringC2EPK20__CFAttributedString + 54
    8   CoreText                            0x000000010a748ae2 CTTypesetterCreateWithAttributedString + 78
    9   QuartzCore                          0x00000001052a4b11 -[CATextLayer _retainTypesetter] + 178
    10  QuartzCore                          0x00000001052a4e44 -[CATextLayer _applyLinesToFunction:info:]     + 69
    11  QuartzCore                          0x00000001052a54b6 -[CATextLayer drawInContext:] + 316
    12  QuartzCore                          0x0000000105290afa _ZN2CA5Layer8display_Ev + 558
    13  QuartzCore                          0x0000000105285c69     _ZN2CA5Layer17display_if_neededEPNS_11TransactionE + 293
    14  QuartzCore                          0x0000000105285cf9     _ZN2CA5Layer28layout_and_display_if_neededEPNS_11TransactionE + 35
    15  QuartzCore                          0x000000010527a475     _ZN2CA7Context18commit_transactionEPNS_11TransactionE + 277
    16  QuartzCore                          0x00000001052a7c0a _ZN2CA11Transaction6commitEv + 486
    17  QuartzCore                          0x00000001052a837c _        ZN2CA11Transaction17observer_callbackEP19__CFRunLoopObservermPv + 92
    18  CoreFoundation                      0x0000000104685367     __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
    19  CoreFoundation                      0x00000001046852d7 __CFRunLoopDoObservers + 391
    20  CoreFoundation                      0x000000010467af2b __CFRunLoopRun + 1147
    21  CoreFoundation                      0x000000010467a828 CFRunLoopRunSpecific + 488
    22  GraphicsServices                    0x000000010a6faad2 GSEventRunModal + 161
    23  UIKit                               0x0000000102cf4610 UIApplicationMain + 171
    24  Baaz                                0x00000001022c589d main + 109
    25  libdyld.dylib                       0x000000010653492d start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by mamouneyya** [comment]
P.S. Here is my code:

```
override func segmentedPager(segmentedPager: MXSegmentedPager, attributedTitleForSectionAtIndex index: Int) -> NSAttributedString {

    var attributes = [String: NSObject]()

    if index < self.titles.count - 1 {
        attributes = [NSFontAttributeName: AppFont.Regular.Size(13), NSForegroundColorAttributeName: UIColor(red: 0.78, green: 0.78, blue: 0.78, alpha: 1.0)]
    } else {
        attributes = [NSFontAttributeName: AppFont.Bold.Size(25), NSForegroundColorAttributeName: UIColor(red: 0.78, green: 0.78, blue: 0.78, alpha: 1.0)]
    }

    return NSAttributedString(string: self.titles[index], attributes: attributes)
}
```

**Comment by maxep** [comment]
Fixed in 3.0.4
